runtime/interpreter/value.go

```diff
@@ -129,6 +134,10 @@ func (BoolValue) SetOwner(_ *common.Address) {
 	// NO-OP: value cannot be owned
 }
 
+func (BoolValue) Modified() bool {
+	return false
```

Comment:
Is this false because these values (as well as integers, etc) have copy semantics?